testData/pkg/apis/fun/v1alpha1/toy_types.go

```diff
@@ -28,6 +28,10 @@ import (
 type ToyS3LogConfig struct {
 	// +kubebuilder:validation:MinLength=1
 	Bucket string `json:"bucket,omitempty"`
+	// Must be an IPv4 or IPv6 address
+	Address string `json:"address"`
+	// Must be a value in range that agrees with type of "Address" (i.e., 1-32 for IPv4, 1-128 for IPv6).
+	Prefix int `json:"prefix"`
```

Comment:
I don't quite understand why ```address``` and ```prefix``` are part of this struct.   Line 89 requires that entries in this array be of string or integer.  (e.g., ```kubebuilder:validation:AnyOf=string,interface```)

Reply:
not quite following this either -- I'd imagine `string,integer` would be for something like port.  I could see `AnyOf` for types that have custom deserialization and look like this (e.g. `AnyOf[]=type:string;format=ipv4` and `AnyOf[]=type:string;format:ipv6`), but this example isn't clear.

Reply:
@alegacy I was having some different interpretation about this annotation, however, @DirectXMan12 cleared my confusion. I will update the code shortly.